Documentation/guides/building-apps/build-properties.md

```diff
@@ -549,6 +549,39 @@ The most common values for this property are:
 
 Added in Xamarin.Android 6.1.
 
+## AndroidIncludeWrapSh
+
+A boolean value which indicates whether the Android wrapper script
+([`wrap.sh`](https://developer.android.com/ndk/guides/wrap-script))
+should be packaged into the APK. The property defaults to `false`
+since the wrapper script may significantly influence the way the
+application starts up and works and the script should be included 
+only when necessary e.g. for debugging or otherwise changing the
+application startup/runtime behavior.
+
+The script is added to the project using the
+[`@(AndroidNativeLibrary)`](./build-items.md#androidnativelibrary)
+build action, because it is placed in the same directory where
+architecture-specific native libraries, and must be named `wrap.sh`.
+The name can be specified using the `Link` metadata of the
+`AndroidNativeLibrary`:
+
+```xml
+<AndroidNativeLibrary Include="/path/to/my/arm64-wrap.sh" Condition=" '$(UseASAN)' != '' ">
```

Comment:
The `Condition` here will likely add more questions than answers; I think it should be removed.